cmd/puma-dev/main_darwin.go

```diff
@@ -59,7 +59,16 @@ func main() {
 	}
 
 	if *fInstall {
-		err := dev.InstallIntoSystem(*fInstallPort, *fInstallTLS, *fDir, *fDomains, (*fTimeout).String())
+		err := dev.InstallIntoSystem(&dev.InstallIntoSystemArgs{
+			ApplinkDirPath:     *fDir,
+			Domains:            *fDomains,
+			LaunchAgentDirPath: "~/Library/LaunchAgents",
+			ListenPort:         *fInstallPort,
+			LogfilePath:        "~/Library/Logs/puma-dev.log",
```

Comment:
probably a good idea to pull this out into shared variable, then add something like `puma-dev -f logs` (similar to `heroku -f logs`)